dataserver.js

```diff
@@ -0,0 +1,29 @@
+'use strict'; // eslint-disable-line strict
+require('babel-core/register');
+
+const arsenal = require('arsenal');
+const config = require('./lib/Config.js').default;
+const logger = require('./lib/utilities/logger').logger;
+
+if (config.backends.data === 'file') {
+    /**
+     * Configure the file paths for data if using the file
+     * backend. If no path provided, uses data at the root of
+     * the S3 project directory.
+     */
+    const dataPath = process.env.S3DATAPATH ?
```

Comment:
I would side with @LaurenSpiegel on this one. Although the dmd may reside on a different server it will still be packaged and deployed using S3/ project. What services we start on independent servers is a different story but one thing is for sure, they should be pulling config from one place. It's easier to manage and we will have no ENV vars floating all over the place. You can still use the ENV vars if you want to provide overrides.